spec/features/user_creates_comment_spec.rb

```diff
@@ -0,0 +1,50 @@
+feature "user creates comment" do
+
+  let(:julie){ Fabricate(:user, name: "Julie") }
+
+  background do
+    bob = Fabricate(:user, name: "Bob")
+    ingrid = Fabricate(:user, name: "Ingrid")
+    Fabricate(:post, author: bob, title: "Bob's Burger Recipe")
+    Fabricate(:post, author: julie, title: "Julie's Intro to XPath", body: "XPath is *great*")
```

Comment:
Ingrid, Julie, and Julie's posts aren't a necessary part of the setup here, since the tests only interact with "Bob's Burger Recipe"